src/Microsoft.TestPlatform.Client/TestPlatform.cs

```diff
@@ -142,7 +142,6 @@ public TestPlatform()
             // sources tells us we should run in-process (i.e. in vstest.console). Because
             // of this no session will be created because there's no testhost to be launched.
             // Expecting a subsequent call to execute tests with the same set of parameters.
```

Comment:
Stale comment. The new one (on the other file) seems to have a little less information but I am not sure if some part are no longer relevant.

Reply:
this looks still relevant. it specifies why the method returns false, whereas the other comment specifies why we need to explicitly send a complete event for failures but not for success